### PR TITLE
JSS to Emotion migration. Focus on parameters

### PR DESCRIPTION
**Pull request by EstherDarkish**

### CODE REVIEW

**Review by trikimahdi — APPROVED** [review]
Style ok
code ok

File: src/components/dialogs/parameters/parameters.js [hunk 1]
```diff
@@ -258,7 +255,9 @@ export const TabPanel = (props) => {
             style={{ flexGrow: 1 }}
             {...other}
         >
-            {(value === index || keepState) && <Box p={1}>{children}</Box>}
+            {(value === index || keepState) && (
+                <Box sx={styles.panel}>{children}</Box>
```
Comment (Tristan-WorkGH) [inline src/components/dialogs/parameters/parameters.js:259]: [The doc](https://mui.com/system/properties/) say that `p` is an abbreviation of `padding`, not margin. Is this wanted ?

File: src/components/dialogs/parameters/parameters.js [hunk 2]
```diff
@@ -81,32 +80,27 @@ import {
...
+export const CloseButton = ({ hideParameters, ...props }) => {
     return (
-        <LabelledButton
-            callback={hideParameters}
-            label={'close'}
-            name={classeStyleName}
-        />
+        <LabelledButton callback={hideParameters} label={'close'} {...props} />
     );
 };
```
Comment (Tristan-WorkGH) [inline src/components/dialogs/parameters/parameters.js:87]: We lost the `styles.button` passed from children who isn't neither passed to `<LabelledButton/>`
```diff
- <CloseButton hideParameters={hideParameters} className={classes.button} />
+ <CloseButton hideParameters={hideParameters} />
```

File: src/components/dialogs/parameters/short-circuit-parameters.js [hunk 3]
```diff
@@ -63,17 +64,12 @@ function getValue(param, key) {
...
 const DropDown = ({ value, label, values, callback }) => {
-    const classes = useStyles();
     return (
         <>
-            <Grid item xs={8}>
-                <Typography component="span" variant="body1">
-                    <Box fontWeight="fontWeightBold" m={1}>
-                        <FormattedMessage id={label} />
-                    </Box>
-                </Typography>
```
Comment (Tristan-WorkGH) [inline src/components/dialogs/parameters/short-circuit-parameters.js:74]: Why did you remove the `<Typography/>` tag here? You didn't remove it in others files.

**Review by Tristan-WorkGH — COMMENTED** [review]
Some questions

File: src/components/dialogs/parameters/short-circuit-parameters.js (hunk 3 above)
Comment (EstherDarkish) [reply src/components/dialogs/parameters/short-circuit-parameters.js:74]: We now use a bold font from the style instead.

**Review by Tristan-WorkGH — APPROVED** [review]
